src/meson.build

```diff
@@ -35,21 +18,14 @@ clippy_sources += gnome.compile_resources(
     c_name: 'clippy'
 )
 
-clippy_lib = shared_library('clippy-' + api_version,
+gtk_modules_path = join_paths(gtk_name,
```

Comment:
In the Flatpak runtime I'm currently using, I don't see this path. I have:
- `/usr/lib/gtk+-3.0/modules` with libcanberra-gtk-module in it
- `/usr/lib/gtk+-3.0/3.0.0/{immodules,printbackends}` but no `modules`

Reply:
Right. But `/usr/lib/gtk+-3.0/3.0.0/modules` is the right one for the `GTK3_MODULES`. It will exist once the runtimes support the extension point (PRs opened upstream).

Reply:
Got it.